src/hotspot/cpu/x86/vm_version_x86.hpp

```diff
@@ -363,7 +365,8 @@ class VM_Version : public Abstract_VM_Version {
     decl(HV,                "hv",                46) /* Hypervisor instructions */ \
     decl(SERIALIZE,         "serialize",         47) /* CPU SERIALIZE */ \
     decl(GFNI,              "gfni",              48) /* Vector GFNI instructions */ \
-    decl(AVX512_BITALG,     "avx512_bitalg",     49) /* Vector sub-word popcount and bit gather instructions */
+    decl(AVX512_BITALG,     "avx512_bitalg",     49) /* Vector sub-word popcount and bit gather instructions */\
+    decl(AVX512_FP16,       "avx512_fp16",       50) /* Vector FP16 instructions*/
```

Comment:
Please also handle the newly added feature in 
src/jdk.internal.vm.ci/share/classes/jdk.vm.ci.amd64/src/jdk/vm/ci/amd64/AMD64.java

Reply:
done